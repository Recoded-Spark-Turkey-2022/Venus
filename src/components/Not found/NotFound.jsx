 import { useEffect } from "react";
 import { useNavigate } from "react-router-dom";
import Container from "../UI/Container";
import notfound from '../../assets/404.png'


const NotFound = ()=>{

     const navigate = useNavigate();
    useEffect(() => {
      setTimeout(() => navigate("/"), 5000);
    }); 

    return(
        
        <Container>
         <div className="flex flex-col justify-center items-center mt-20">
        <h1 className="font-extrabold text-4xl md:text-6xl mt-10 text-openRose text-center">PAGE NOT FOUND</h1>
        <p className="mt-5 text-darkGrey text-lg md:text-xl text-left md:text-center">Sorry, the page you are looking for doesn&apos;t exist.</p>
        <p className="text-darkGrey text-lg md:text-xl text-left md:text-center">Don&apos;t worry, you will be directed to the home page in 5 seconds.</p>
        <img className="w-full md:w-1/2" src={notfound} alt='404'/>
        </div>
        </Container>
        

    )
}

export default NotFound;