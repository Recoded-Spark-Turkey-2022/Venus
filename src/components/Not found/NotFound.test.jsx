import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NotFound from './NotFound'


it(`renders correctly` , () => {
    const tree = renderer.create(
    <Router>
        <NotFound />
    </Router>);
    expect(tree).toMatchSnapshot();
});