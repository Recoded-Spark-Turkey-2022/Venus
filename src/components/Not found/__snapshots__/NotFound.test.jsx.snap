// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container mx-auto px-10"
>
  <div
    className="flex flex-col justify-center items-center mt-20"
  >
    <h1
      className="font-extrabold text-4xl md:text-6xl mt-10 text-openRose text-center"
    >
      PAGE NOT FOUND
    </h1>
    <p
      className="mt-5 text-darkGrey text-lg md:text-xl text-left md:text-center"
    >
      Sorry, the page you are looking for doesn't exist.
    </p>
    <p
      className="text-darkGrey text-lg md:text-xl text-left md:text-center"
    >
      Don't worry, you will be directed to the home page in 5 seconds.
    </p>
    <img
      alt="404"
      className="w-full md:w-1/2"
      src="404.png"
    />
  </div>
</div>
`;
